models/_sector/nft/trades/solana/nft_solana_trades.sql

```diff
@@ -0,0 +1,72 @@
+{{
+    config(
+        schema = 'nft_solana'
+        , tags = ['dunesql']
+        , alias = alias('trades')
+        , materialized = 'view'
+        ,post_hook='{{ expose_spells(\'["solana"]\',
+                                    "sector",
+                                    "nft",
+                                    \'["ilemi"]\') }}'
+    )
+}}
+
+{% set solana_marketplaces = [
+    ref('magiceden_v2_solana_trades')
+    , ref('tensorswap_v1_solana_trades')
+    , ref('tensorswap_v2_solana_trades')
+] %}
+
+
+{% for marketplace in solana_marketplaces %}
+SELECT
+        blockchain,
+        project,
+        version,
+        cast(date_trunc('day', block_time) as date) as block_date,
+        cast(date_trunc('month', block_time) as date) as block_month,
```

Comment:
in the future, when we finalize strategy for partitioning, we may end up moving these to project-level spells so they can partition on one or the other, but this is fine for now